Protection Profile/BiocPP.adoc

```diff
@@ -413,16 +413,16 @@ This section lists SFRs for the biometric enrolment and verification.
 
 *FIA_MBV_EXT.1.2*:: The TSF shall provide a biometric verification mechanism with the [*selection*: _FMR, FAR_] not exceeding [*assignment*: _defined value_] and [*selection*: _FNMR, FRR_] not exceeding [*assignment*: _defined value_].
 
-*Application Note {counter:remark_count}*:: If the TOE supports multiple modalities, the ST author may iterate the SFR to define different error rates for each modality.
+*Application Note {counter:remark_count}*:: If the TOE supports multiple modalities, the error rates do not to be identical.
```

Comment:
I'm not sure if we need to specify whether this requirement can be iterated or not. That is the norm for having different answers (since you can define the assignment as to a specific option), but I'm not sure if that convention is needed or not.

@n-kai what do you think?

Reply:
@woodbe 
I agree with you. It's better to remove the last application note (I checked MDF PP and related PP modules but no one has such application note)

Reply:
I have removed this application note in a39029230239343e07bcf98b9a5b488e6c88cb92. Should we also remove the next note that seems to also correspond to iteration?

> If the TOE supports multiple modalities, not all modalities need to define the Evaluation Activities.

Reply:
OK, I'll remove it

Reply:
@gfiumara yes, I would remove that one too